src/util/dialog/DEPromptDialog.js

```diff
@@ -0,0 +1,109 @@
+/**
+ *  @author sriram
+ *
+ * Rewrite IPlantPromptDialog in react.
+ **/
+
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+
+import intlData from "./messages";
+import DEDialogHeader from "./DEDialogHeader";
+import withI18N, { getMessage } from "../I18NWrapper";
+
+import Button from "../../../node_modules/@material-ui/core/Button/Button";
```

Comment:
Looks like a couple of messed up imports in this component:
```suggestion
import Button from "@material-ui/core/Button";
```